src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -54,8 +54,9 @@
     /**
      * The command for fetching projects.
      */
-    public static final String GERRIT_LS_PROJECTS = "gerrit ls-projects";
+    public static final String GERRIT_LS_PROJECTS = "gerrit ls-projects --prefix ";
```

Comment:
Note: The `--prefix` option is not documented on the `ls-projects` command for 2.14, 2.15 or 2.16.

Reply:
True, this needs to be fixed in Gerrit though.

With regards to the support in Gerrit < 2.13, needs to be documented in this plugin and checked as soon as the version is queried.

Reply:
I've posted the documentation for review https://gerrit-review.googlesource.com/c/gerrit/+/212012